content/asset-tracking/using-the-sdks.textile

```diff
@@ -181,13 +235,8 @@ The following code example demonstrates initializing and starting the subscriber
 // Initialise and Start the Subscriber
 val subscriber = Subscriber.subscribers() // Get an AssetSubscriber
     .connection(
-                ConnectionConfiguration(
-                    Authentication.basic(
-                        CLIENT_ID,
-                        ABLY_API_KEY,
-                    )
-                )
-            ) // provide Ably configuration with credentials
+        ConnectionConfiguration(Authentication.basic(CLIENT_ID, ABLY_API_KEY))
+    ) // provide Ably configuration with credentials
```

Comment:
Given that in the `.resolution` from the line below the comment is after the opening bracket, maybe it should look the same in the case of `.connection`? :thinking:
```kotlin
.connection( // provide Ably configuration with credentials
    ConnectionConfiguration(Authentication.basic(CLIENT_ID, ABLY_API_KEY))
)
```

Reply:
Good spot!